### PR TITLE
Added the autotagging Jenkins job for psap, perf-scale

### PR DESCRIPTION
**Pull request by athiruma**

### CODE REVIEW

File: cloud_governance/tag_non_cluster/run_tag_non_cluster_resources.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def tag_na_resources(file_name: str, file_path: str = '/tmp/', tag_operation: st
     @param file_name:
     @return:
     """
-    file_name = f'{file_path}{region}-{file_name}'
+    file_name = f'{file_path}{file_name}'
```
Comment (ebattat) [inline cloud_governance/tag_non_cluster/run_tag_non_cluster_resources.py:53]: Why did u remove the region ?

File: jenkins/run_policies.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def get_custodian_policies(type: str = None):
...
             os.system(f"sudo podman run --rm --name cloud-governance -e account='psap' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PSAP} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PSAP} -e AWS_DEFAULT_REGION={region} -e dry_run=no -e policy_output=s3://{BUCKET_PSAP}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
-            os.system(f"sudo podman run --rm --name cloud-governance -e account='perf-scale' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE} -e AWS_DEFAULT_REGION={region} -e dry_run=yes -e policy_output=s3://{BUCKET_PERF_SCALE}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
+            os.system(f"sudo podman run --rm --name cloud-governance -e account='perf-scale' -e policy={policy} -e AWS_ACCESS_KEY_ID={AWS_ACCESS_KEY_ID_DELETE_PERF_SCALE} -e AWS_SECRET_ACCESS_KEY={AWS_SECRET_ACCESS_KEY_DELETE_PERF_SCALE} -e AWS_DEFAULT_REGION={region} -e dry_run=no -e policy_output=s3://{BUCKET_PERF_SCALE}/{LOGS}/{region} -e log_level=INFO quay.io/ebattat/cloud-governance:latest")
```
[the remark this answers is not in the document]
Comment (athiruma) [reply jenkins/run_policies.py:50]: No

File: cloud_governance/tag_non_cluster/run_tag_non_cluster_resources.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/tag_non_cluster/run_tag_non_cluster_resources.py:53]: It causes noise. When giving filename.
Ex: the generated filename is us-west-1-tag_user.csv
given filename is us-west-1-tag_user.csv. It added us-west-1-us-west-1-tag_user.csv. So I removed it.

**Review by ebattat — COMMENTED** [review]
/LGTM

**Pull request merged**